app/controllers/display_controller.rb

```diff
@@ -75,6 +75,48 @@ def read_all_works
     session[:col_id] = @collection.slug
   end
 
+  def display_page
+    # Set meta information for web crawlers and archival
+    @page_title = "#{@page.title || "Page #{@page.position}"} - #{@work.title} - #{@collection.title}"
+    @meta_description = "Transcript of #{@page.title || "page #{@page.position}"} from #{@work.title} in the #{@collection.title} collection."
+    @meta_keywords = [@work.title, @collection.title, @page.title, "transcript", "transcription", "historical document"].compact.join(", ")
+    
+    # Generate structured data for better content understanding
+    @structured_data = {
+      "@context" => "https://schema.org",
+      "@type" => "DigitalDocument",
+      "name" => @page.title || "Page #{@page.position}",
+      "description" => @meta_description,
+      "text" => @page.source_text,
```

Comment:
The contents of `source_text` can contain XML or wiki-text mark-up and can be very long.  If this is not going to be valid structured data, we should use the `verbatim_transcription_plaintext` method here.